.github/scripts/format-check

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eux
+
+unformatted() { 
+    set -x
+    make format
+    test -z "$(git status --porcelain)"
+}
+
+export -f unformatted
+
+source conda/etc/profile.d/conda.sh
+conda activate
```

Comment:
Looking at the Actions output, I recall that we should turn off tracing around conda commands -- their trace is just too messy. Could you change these two lines to
```
set +x
source conda/etc/profile.d/conda.sh
conda activate
set -x
```
? That'll clean things up nicely.

Reply:
This is superseded by my Slack comment, which simplifies this script further.